src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -92,6 +92,8 @@
 GENOME_ATTR_TOOLS = ['checkm2', 'gtdb_tk']
 # tool result will be parsed as heatmap data
 HEATMAP_TOOLS = ['microtrait']
+# tools result will be checked for fatal error files
+CHECK_FATAL_ERROE_TOOLS = ["gtdb_tk", "microtrait"]
```

Comment:
Just check all of them.

Reply:
Are we extending to checkm2 and mash as well? Bad genomes in checkm2 result might not be that obvious.

Reply:
We should run through their folders in case we need to add fatal error files later

Reply:
I didn't realize there was no list of the supported tools and no way to easily compute it. Given that, I'd get rid of this field and add something like

```python
ALL_TOOLS = GENOME_ATTR_TOOLS + HEATMAP_TOOLS + ["mash"]
```

Longer term I'm thinking we should refactor the parser to 
* push as much processing as possible back to the individual tool code
* Implement the method running with a dict of tool name to parser method or something like that. The trick there is that the GTDB and CheckM output has to end up in the same file...

but that's later

Reply:
Updated!

Reply:
:+1: